projects/azure/azure-cicd/script-cm.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: token-refresh-script
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    export KUBECONFIG=/tmp/registry
+    oc login --username ${USERNAME} --password ${PASSWORD} --server console.reg-aws.openshift.com > /dev/null
+    TOKEN=$(oc whoami -t)
+    export KUBECONFIG=/var/lib/origin/openshift.local.config
+    oc delete secret aws-reg-token
```

Comment:
`oc create secret generic aws-reg-token --from-literal=token=${TOKEN} -n azure --dry-run -o yaml | oc apply -f -` and you don't need the two following commands.

Reply:
The funny thing is it fails with oc permission denied error :D but I did this late in the day so will test again.

Reply:
You probably need to `unset KUBECONFIG` to force using the service account which has the correct permissions.